tutorials/cp-mobile-dev-kit-create-customer/cp-mobile-dev-kit-create-customer.md

```diff
@@ -10,10 +10,11 @@ author_profile: https://github.com/jitendrakansal
 ---
 
 ## Prerequisites
-- You have created an MDK app as described in [Get Started with the Mobile Development Kit](https://developers.sap.com/mission.mobile-dev-kit-get-started.html).
+- You have created an MDK app as described in [Create Your First Mobile App with the Mobile Development Kit (MDK)](mission.mobile-dev-kit-get-started).
 
 ## Details
-**Note**: Ensure that you have created an MDK app as described in [Get Started with the Mobile Development Kit](https://developers.sap.com/mission.mobile-dev-kit-get-started.html).
```

Comment:
Move this and all extra text after  You will learn section

Reply:
I am working with Daniel.
Thanks for making the changes, everything seems great, but one more thing, please move the note also to after "you will learn".

Reply:
@MichaelCzcz i have removed the "note" part as this is already covered in the pre-requisite. Pls have a look and let me know if something is missing.